rel-eng/tito.props

```diff
@@ -546,9 +546,9 @@ whitelist =
   tracer
   yggdrasil
 
-[foreman-client-nightly-rhel7]
+[foreman-client-3.4-rhel7]
 disttag = .el7
-# don't whitelist foreman-release here, it's already in foreman-nightly-nonscl-rhel7
+# don't whitelist foreman-release here, it's already in foreman-3.4-nonscl-rhel7
```

Comment:
now that you say it, there is no 3.4-nonsc-7, so you NEED to whitelist it here!

same should happen for nightly, I geuss

Reply:
done